docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Pod Memory and CPU Resources"
+date: 2020-06-30T08:55:00-05:00
+draft: false
```

Comment:
Placeholder request for @rosemarymarano to please suggest description and weight that fits in with the updates you did for other FAQ's.

Reply:
Yes. I'll provide that input on my final review of the FAQ.